css/properties/align-items.json

```diff
@@ -8,13 +8,34 @@
             "webview_android": {
               "version_added": "4.4"
             },
-            "chrome": {
-              "version_added": "21",
-              "notes": "Older versions of the specification treat absolute positioned children as though they are a 0 by 0 flex item. Later specification versions take the children out of the flow and set their positions based on align and justify properties. Chrome implements the new behavior beginning with Chrome 52."
-            },
-            "chrome_android": {
-              "version_added": null
-            },
+            "chrome": [
+              {
+                "version_added": "52"
+              },
+              {
+                "version_added": "29",
+                "partial_implementation": true,
+                "notes": "Older versions of the specification treat absolute positioned children as though they are a 0 by 0 flex item. Later specification versions take the children out of the flow and set their positions based on align and justify properties. Chrome implements the new behavior beginning with Chrome 52."
+              },
+              {
+                "prefix": "-webkit-",
```

Comment:
@Elchi3 Do you have a preference on whether 'webkit' includes dashes. I just grepped my local repo and found we're pretty inconsistent.

Reply:
Is that in the `css` tree or the `api` tree?

As the `css` tree should contain dashes, whereas the `api` tree should not (see the [Vendor Prefix Glossary page](https://developer.mozilla.org/docs/Glossary/Vendor_Prefix) for more information).

Reply:
I did not know that. (I'm only every removing prefixes.) The css/ folder appears to be consistent. The api/ folder is not. I didn't check the others.

Reply:
What @ExE-Boss says is correct. Not sure if we could validate some more automatically there.